core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -112,22 +114,6 @@ class CloudSqlInstance {
     }
   }
 
-  static long secondsUntilRefresh(Date expiration) {
-    Duration timeUntilExp = Duration.between(Instant.now(), expiration.toInstant());
```

Comment:
This is probably better than the long-based calculation. Let's use that below.

Reply:
Fixed.